src/main/java/com/github/danielflower/mavenplugins/gitlog/GenerateMojo.java

```diff
@@ -187,6 +199,18 @@
 	 */
 	@Parameter
 	private String path;
+	
+	/**
+	 * Include in the changelog the commits after the last release
+	 */
+	@Parameter(defaultValue = "false")
+	private boolean includeOnlyCommitsAfterRelease;
+	
+	/**
+	 * the tag name of release

```

Comment:
Can you explain a bit more about this option? Is it the prefix of that tag name, or a git commit, that marks a commit as a release?